airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -344,6 +345,10 @@ public String getSpecCacheBucket() {
     return getEnvOrDefault(SPEC_CACHE_BUCKET, DEFAULT_SPEC_CACHE_BUCKET);
   }
 
+  public Optional<String> getLocalCatalogPath() {
+    return Optional.ofNullable(getEnv(LOCAL_CATALOG_PATH));
+  }
```

Comment:
Someone on the platform team can correct me, but I don't think we should be adding new things to EnvConfigs as it should be on its way out, replaced by the micronaut configuration.

Reply:
so the reason I went this route is as I understand micronaut config is not available at the gradle task level. Was I wrong there?

Reply:
Context here, after a conversation with Pedro, and some digging on my side we're going to leave this as EnvConfig until after the monorepo switch. Once complete there's likely some improvements to be done about how LocalDefinitions is loaded, which should open up the possibility of using micronauts value decorators